test/test_base_providers.py

```diff
@@ -57,14 +63,17 @@ def setUp(self) -> None:
         self.base_warehouse = DataWarehouse(self.base_company_data, self.base_production_bm, self.base_EI_bm)
         self.company_ids = ["US0079031078",
                             "US00724F1012",
-                            "FR0000125338"]
+                            "FR0000125338",
```

Comment:
A note for the future: this is an example of test data that we should re-sync with our sample public dataset.  It's not helpful, as a test case, to have random ISINs and random production, emission, and intensity numbers that don't tie back to anything meaningful, unless the test data has been carefully constructed to trigger a particular code path.  This is not an example of that; in the future we should fix this bit of code we inherited.  Issue filed: https://github.com/os-climate/ITR/issues/167